android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/components/LocationFilter.kt

```diff
@@ -68,111 +70,124 @@ fun LocationFilter(
 		titleId = R.string.location,
 		closeAction = closeAction,
 		content = {
-			RangeSlider(
+			Column(
```

Comment:
Wrapping content with a column, everything else remains the same.

Reply:
Just out of curiosity, but why are we wrapping everything in a column?
I'm guessing that Compose is being weird and it fixes something #GottaLoveCompose

Reply:
Oh, I just made a bad decision of putting the column in the defaultFilter component. And I can't doing anything to change the configuration for that column, so I move it out.